api/src/services/cereNetworkService.js

```diff
@@ -50,16 +61,15 @@ function initFaucet(mnemonic) {
   return newPair;
 }
 
-init();
-
 module.exports = {
+  init,
   supportsNetwork: (network) => {
     return networkParams.has(network.toUpperCase());
   },
   getBalance: async (network, address) => {
     const { api, _ } = networkParams.get(network.toUpperCase());
     const { nonce, data: balance } = await api.query.system.account(address);
-    return balance.free;
+    return balance.free.toString();
```

Comment:
Let's return BigNumber with a value converted to decimal.

Reply:
I suggest using everywhere strings as an interface. `BigNumber` is a wrap required for mathematical operations and string is a more universal format that can be used to create `BigNumber`, send it to the client, and store it in the database.

Reply:
Discussed. Let's use BN.js

Reply:
resolved